gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail.component.ts

```diff
@@ -13,23 +13,51 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Component, Inject, OnDestroy, OnInit } from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
 
+import { UIRouterStateParams } from '../../../../ajs-upgraded-providers';
+import { User } from '../../../../entities/user/user';
 import { UsersService } from '../../../../services-ngx/users.service';
 
+interface UserDisplayable {
```

Comment:
Maybe it's more clear to create a view model interface that extends User
```suggestion 
interface UserVM extends User {
```